oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/stream/BanyanDBSpanAttachedEventQueryDAO.java

```diff
@@ -51,11 +51,11 @@ public BanyanDBSpanAttachedEventQueryDAO(BanyanDBStorageClient client) {
     }
 
     @Override
-    public List<SpanAttachedEventRecord> querySpanAttachedEvents(SpanAttachedEventTraceType type, String traceId) throws IOException {
+    public List<SpanAttachedEventRecord> querySpanAttachedEvents(SpanAttachedEventTraceType type, List<String> traceIds) throws IOException {
         final StreamQueryResponse resp = query(SpanAttachedEventRecord.INDEX_NAME, TAGS, new QueryBuilder<StreamQuery>() {
             @Override
             protected void apply(StreamQuery query) {
-                query.and(eq(SpanAttachedEventRecord.RELATED_TRACE_ID, traceId));
+                query.and(having(SpanAttachedEventRecord.RELATED_TRACE_ID, traceIds));
```

Comment:
@hanahmily @lujiajing1126 Should we use `have` or `or`? Or the latest `in`?

Reply:
I think `have` means having all conditions. @mrproliu Please check through e2e or locally

Reply:
`in` should be a right term, https://github.com/apache/skywalking/pull/10181

Reply:
I have tested locally, the `have` only matches the first trace. I think I need to wait for the `in` condition. The `or` condition looks like it needs all the conditions to be `or` to be valid.

Reply:
You put one trace ID or all to query? I think if we use multiple trace IDs, having would match nothing.

Reply:
Generally, we should use one query to get all events for all traces once.

Reply:
I put all trace IDs to the query, It matches the first trace(in the ID list) result.

Reply:
> Generally, we should use one query to get all events for all traces once.

Yes, so I think I need to wait for the `in` condition. Also, I found two issues when storing and querying. 
1. The `sampled_slow_trace_record` cannot store the `latency` field, It throws `invalid ref to tag latency`.
2. The `span_attached_event_record` fields `timestamp` is missing when query.

Reply:
@lujiajing1126 Could you follow this?

Reply:
> @lujiajing1126 Could you follow this?

Oh, I didn't notice this PR.

> @hanahmily @lujiajing1126 Should we use `have` or `or`? Or the latest `in`?

From the semantics, it should be `in` here. `having` means subset, usually for `Set`/`List` typed tag.